File: update_colors.js
const fs = require('fs');
const path = '/Users/alex/Desktop/7_24bets-landing-page/components/VIPRafflePromo.tsx';
let content = fs.readFileSync(path, 'utf8');

// Colors replacement map
const replacements = [
    [/amber-100/g, 'cyan-100'],
    [/amber-200/g, 'cyan-200'],
    [/amber-300/g, 'cyan-300'],
    [/amber-400/g, 'cyan-400'],
    [/amber-500/g, 'cyan-500'],
    [/amber-600/g, 'cyan-600'],
    [/amber-700/g, 'cyan-700'],
    [/text-amber-950/g, 'text-[#0A0D14]'],
    [/yellow-400/g, 'cyan-400'],
    [/yellow-500/g, 'cyan-500'],
    [/yellow-600/g, 'cyan-600'],
    [/rgba\(245,166,35,/g, 'rgba(0,229,255,'],
    [/rgba\(251,191,36,/g, 'rgba(0,229,255,'],
    [/rgba\(212,175,55,/g, 'rgba(0,229,255,'],
    [/rgba\(250,204,21,/g, 'rgba(0,229,255,'],
    [/gold-gradient-text/g, 'text-transparent bg-clip-text bg-gradient-to-r from-[#00E5FF] to-cyan-300'],
    // Make the main action button use the #00E5FF cyan
    [/bg-cyan-400/g, 'bg-[#00E5FF]'], // just in case
];

for (const [regex, replacement] of replacements) {
    content = content.replace(regex, replacement);
}

fs.writeFileSync(path, content, 'utf8');
console.log('Colors updated successfully in VIPRafflePromo.tsx');

File: update_colors.py
import os
import glob

files = glob.glob('components/*.tsx') + ['App.tsx', 'index.css']

# We want to replace backgrounds:
# bg-[#050505] -> bg-[#0A0C10]
# bg-[#06080C] -> bg-[#0A0C10] 
# bg-[#0a0d14] -> bg-[#0A0C10]
# bg-[#0A0D14] -> bg-[#0A0C10]
# bg-[#1b2230] -> bg-[#0A0C10]
# bg-[#111111] -> bg-[#0A0C10]
# bg-[#1b2230] -> bg-[#0A0C10]
# bg-black -> bg-[#0A0C10] (but maybe keep bg-black/50, so use regex)

import re

for filepath in files:
    if not os.path.isfile(filepath): continue
    with open(filepath, 'r') as f:
        content = f.read()
    
    # Backgrounds
    content = re.sub(r'bg-\[#(050505|06080C|0a0d14|0A0D14|1b2230|111111)\]', 'bg-[#0A0C10]', content)
    # Be careful with bg-black, only replace if it's exactly bg-black (not bg-black/50)
    content = re.sub(r'\bbg-black\b(?!\/)', 'bg-[#0A0C10]', content)
    
    # Also we need to strip gold/green
    # text-emerald-500, text-emerald-400 -> text-[#00E5FF]
    # text-[#FFD700], text-yellow-400, text-amber-400 -> text-white
    # bg-emerald-500, bg-[#00E676] -> bg-[#00E5FF]
    
    content = re.sub(r'text-emerald-(400|500)', 'text-[#00E5FF]', content)
    content = re.sub(r'bg-emerald-(400|500)', 'bg-[#00E5FF]', content)
    content = re.sub(r'text-\[#00E676\]', 'text-[#00E5FF]', content)
    content = re.sub(r'bg-\[#00E676\]', 'bg-[#00E5FF]', content)
    
    content = re.sub(r'text-yellow-(400|500)', 'text-zinc-300', content)
    content = re.sub(r'text-amber-(400|500)', 'text-zinc-300', content)
    content = re.sub(r'text-\[#FFD700\]', 'text-white', content)
    
    with open(filepath, 'w') as f:
        f.write(content)
print("Updated colors.")
